server/application/rest/service_shifts.py

```diff
@@ -5,14 +5,25 @@
 import json
 from flask import Blueprint, request, Response
 from flask_cors import cross_origin
-from use_cases.add_service_shifts import shift_add_use_case
-from use_cases.list_service_shifts_use_case import service_shifts_list_use_case
-from repository.mongo.service_shifts import ServiceShiftsMongoRepo
-from domains.service_shift import ServiceShift
+
+# Application-specific imports grouped by package
 from application.rest.status_codes import HTTP_STATUS_CODES_MAPPING
+from application.rest.auth_utils import get_user_email_from_token
```

Comment:
`get_user_from_token` is defined in `server/authentication/authenticate_user.py`. You'll want to import that function in each file and use it. Note that it's `get_user_from_token`, not `get_user_email_from_token`